### DISCUSSION TITLE
OWT Media Server has Congestion control and Bandwidth estimation?

### DISCUSSION THREAD

**Discussion by ihorafanasiev** [open-webrtc-toolkit/owt-server#1371] [Q&A] [answered]
I need some analytic information from OWT Media Server. I want to take information about support Congestion control and Bandwidth estimation. Does  OWT Media Server support this functionality? If yes, how can I get this information? I didn't find any information in the documentation.

Thanks

**Comment by starwarfan** [comment, marked as answer]
OWT server uses sender side bandwidth estimation (transport-cc) from WebRTC stack.
For client publication, you can use getStats function (see client SDK) and get the `availableOutgoingBitrate` from the returned key values.
For client subscription, you can only check bandwidth related stats or modify code in server side as `getEstimatedBandwidthBps`.
